examples/Test Output/Exporters/Custom Exporters/Basic/test_plan.py

```diff
@@ -74,7 +74,19 @@ def __init__(self, file_path):
     def get_text_content(self, source):
         raise NotImplementedError
 
-    def export(self, source: TestReport) -> Optional[str]:
+    def export(
+        self,
+        source: TestReport,
+        export_context: ExportContext,
+    ) -> ExporterResult:
```

Comment:
The change in signature reminds me that exporters are user facing and they may have their own customizations. Do we know the changes may affect these users? If yes, I suggest we make note of this change and put together some doc.

Reply:
It shouldn't be breaking. The case when the user uses a custom exporter which does not implement the new features is handled and will show a warning message for now. (This will be removed in the future).
Creating a doc describing the changes is a good idea!